### ISSUE TITLE
[Edit existing device] Select device combo box is a required field but missing aria-required

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6405]
## Area or Component

Support Admin > Edit existing device > Select device combo box

## Quick Summary

Select device combo box should have `aria-required=true`

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing

## Action Requested

Add `required=true` to the `inputProps` in [the combo box](https://github.com/trussworks/react-uswds/blob/4550708a95f955e1585d05aaf44b3592049d49b8/src/components/forms/ComboBox/ComboBox.tsx#L53)

## Source Code (optional)

[which file or line is the issue attached to?]: #
